### ISSUE TITLE
[BUG] [URGENT] - I can't login to my account (Organisation billing period is not set)

### ISSUE THREAD

**Issue by pierrejo** [formbricks/formbricks#2789]
### Issue Summary

Since i've migrate to 2.2.1 :

```
Error: Organization billing period start is not set
    at E.revalidate (/home/nextjs/apps/web/.next/server/chunks/8763.js:1:3117)
    at async o (/home/nextjs/apps/web/.next/server/chunks/3751.js:1:389)
    at async /home/nextjs/apps/web/.next/server/chunks/1334.js:1:5470
    at async /home/nextjs/apps/web/.next/server/chunks/3751.js:1:479
    at async Promise.all (index 1)
    at async I (/home/nextjs/apps/web/.next/server/chunks/2693.js:1:33552) {
  digest: '602769226'
```
I can't connect to my account (free account with self-hosted instance)

### Steps to Reproduce

just try to login my app

### Expected behavior

can login

### Environment

- [ ] Formbricks Cloud (app.formbricks.com)
- [X] Self-hosted Formbricks

### Desktop (please complete the following information)

```markdown
Docker Formbrick Version : 2.2.1
```

**Comment by Nadyaducalis** [comment]
Have the same issue

**Comment by sindivinch** [comment]
Same issue.

**Comment by mishlaev-personal** [comment]
We are experience the same issue

**Comment by elledanili** [comment]
We are having the same issue

**Comment by lnghost** [comment]
Fix it please

**Comment by mattinannt** [comment]
@lnghost Thanks for pointing that out. We are currently working on a solution and will push a 2.2.2 release with a fix very soon.

**Comment by mattinannt** [comment]
We just released Formbricks 2.2.2 with a fix for this:
https://github.com/formbricks/formbricks/releases/tag/v2.2.2

The Docker images are currently building for the release and should be online in a few minutes. If you are upgrading from Formbricks 2.2 or 2.2.1 please make sure to run the data migration for `2.2` again to fix the issues :-)

**Comment by pierrejo** [comment]
It's **OK** for me :heavy_check_mark:  !  

Thanks for the fix
